util/trace_replay.cc

```diff
@@ -182,7 +194,8 @@ Status Replayer::Replay() {
     }
 
     std::this_thread::sleep_until(
-        replay_epoch + std::chrono::microseconds(trace.ts - header.ts));
+        replay_epoch +
+        std::chrono::microseconds((trace.ts - header.ts) / fast_forward_));
```

Comment:
Do users mostly care about speeding up the replay by a certain ratio, or are they mostly interested in replaying as fast as possible? I believe it might be the later. 
If this is the case, why even take a ratio as input? It could be better to take a boolean input to speed up replay, and wrap the sleep_until inside an if-condition. (If so, `Replay` function itself could take a default boolean arg). 

What do you think?

Reply:
I think user may prefer to have the flexibility to decide the intensiveness of the workload when they start to replay (e.g., the same workload with different average QPS). If user want to do the stress test, we can allow the user specify the flag to a special value (e.g., -1), so replay can jump to the non sleep_until branch.